perf/src/sigverify.rs

```diff
@@ -113,6 +122,66 @@ fn verify_packet(packet: &mut Packet) {
         return;
     }
 
+    if reject_non_vote {
```

Comment:
I think we want this check in `get_packet_offsets` to ensure that the gpu sigverify path hits this code too